addon/services/page-title-list.js

```diff
@@ -189,12 +190,12 @@ export default Service.extend({
       return frontGroups.concat(
         groups.reduce((E, group) => E.concat(group), [])
       );
-    }
+    },
   }),
 
   scheduleTitleUpdate() {
-    let router = getOwner(this).lookup('router:main');
-    let { activeTransition } = router._routerMicrolib;
+    // eslint-disable-next-line ember/no-private-routing-service
+    let { activeTransition } = this.router._router._routerMicrolib;
```

Comment:
@rwjblue any way to get the `activeTransition` without private APIs?

Reply:
Should be fixed with a rebase now